assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss

```
@@ -113,5 +113,12 @@
 				font-weight: $fw-medium;
 			}
 		}
+
+		.googlesitekit-km-widget-tile__table-plain-text {
+			color: $c-surfaces-on-surface;
+			font-size: $fs-label-sm;
```

Could we use the Figma token here?

```suggestion
			font-size: $fs-body-sm;
```

---

Updated ✔️